### PR TITLE
Feat stripeintegration

### PR DESCRIPTION
**Pull request by matoszz**

### CODE REVIEW

File: apps/console/src/app/(protected)/organization-settings/billing-usage/pricing/page.tsx [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-import PricingPage from '@/components/pages/protected/organization/billing/pricing-table'
+import StripePricingTable from '@/components/pages/protected/organization/billing/pricing-table'
 
 const Page: React.FC = () => {
   return (
     <>
     <PageHeading heading="Pricing Plans" eyebrow="Billing" />
-    <PricingPage />
+    <StripePricingTable 
+      pricingTableId="prctbl_1QKO15Bvxky1R7SvxYRBAjs7"
```
Comment (golanglemonade) [inline apps/console/src/app/(protected)/organization-settings/billing-usage/pricing/page.tsx:10]: these variables aren't used and shouldn't be passed in

File: apps/console/src/components/pages/protected/organization/billing/pricing-table.tsx [hunk 2]
```diff
@@ -9,14 +11,62 @@ declare global {
...
-    <stripe-pricing-table
-      pricing-table-id={stripePricingTableId}
-      publishable-key={stripePublishableKey}
-    >
-    </stripe-pricing-table>
-  );
+interface StripePricingTableProps {
+  pricingTableId: string;
+  publishableKey: string;
 }
```
Comment (golanglemonade) [inline apps/console/src/components/pages/protected/organization/billing/pricing-table.tsx:18]: you don't need this

File: apps/console/src/components/pages/protected/organization/billing/pricing-table.tsx [hunk 3]
```diff
@@ -9,14 +11,62 @@ declare global {
...
 
-export default PricingPage;
\ No newline at end of file
+const StripePricingTable: React.FC<StripePricingTableProps> = ({
+  pricingTableId,
+  publishableKey,
+}) => {
```
Comment (golanglemonade) [inline apps/console/src/components/pages/protected/organization/billing/pricing-table.tsx:22]: these aren't used, there is an import at the top:
```
import { stripePricingTableId, stripePublishableKey } from '@repo/dally/auth';
```

that are used instead

```suggestion
const StripePricingTable: React.FC= ({}) => {
```

File: apps/console/src/app/(protected)/organization-settings/billing-usage/pricing/page.tsx (hunk 1 above)
Comment (matoszz) [reply apps/console/src/app/(protected)/organization-settings/billing-usage/pricing/page.tsx:10]: if i remove them then the table doesn't function?

**Pull request closed without merging**